### PR TITLE
fix docker-quickstart.mdx doc

### PR DESCRIPTION
**Pull request by pops-one**

### CODE REVIEW

File: docs/operate/docker-quickstart.mdx [hunk 1]
```diff
@@ -82,18 +82,50 @@ You should change the service name, db path, and config file locations to match
...
 DB_PATH=./ethereum/hyperlane_db
-METRICS_PORT=9090 # Default is 9090, should match port set in config.json
+METRICS_PORT=9090 # external port, default is 9090 and should be different for each validator ie 9091, 9092, ...
 CONFIG_FILE=./ethereum/config.json
 SERVICE_NAME=ethereum
 ```
 
 ## 5. Configure Docker Compose (docker-compose.yml)
 
 ```yaml
```
Comment (paulbalaji) [inline docs/operate/docker-quickstart.mdx:92]: how do you feel about minimising the .env file to just
```env
AWS_ACCESS_KEY_ID=<Your AWS access key ID>
AWS_SECRET_ACCESS_KEY=<Your AWS secret access key>
SERVICE_NAME=ethereum
```

and streamlining the compose file by using the service name to infer the configs?
```yaml
services:
  ethereum:
    <<: *common-validator
    ports:
      - "9091:9090/tcp"

  optimism:
    <<: *common-validator
    ports:
      - "9092:9090/tcp"

  base:
    <<: *common-validator
    ports:
      - "9093:9090/tcp"

x-common-attributes: &common-validator
  image: gcr.io/abacus-labs-dev/hyperlane-agent:7a8478b-20240703-113821
  command: ./validator
  container_name: ${SERVICE_NAME}-validator
  environment:
    CONFIG_FILES: /config.json
    AWS_ACCESS_KEY_ID: ${AWS_ACCESS_KEY_ID}
    AWS_SECRET_ACCESS_KEY: ${AWS_SECRET_ACCESS_KEY}
  configs:
    - source: ./${SERVICE_NAME}/config.json
      target: /config.json
  volumes:
    - ./${SERVICE_NAME}/hyperlane_db:/mnt/hyperlane_db
  restart: unless-stopped
```

File: docs/operate/docker-quickstart.mdx [hunk 2]
```diff
@@ -147,13 +177,15 @@ Bring the containers up:
...
+docker compose --env-file .env.base up validator3 -d
 ```
 
 Ensure there are no errors:
 
 ```bash
-docker logs -f ethereum-validator
+docker logs -f validator1
+docker logs -f validator2
+docker logs -f validator3
```
Comment (paulbalaji) [inline docs/operate/docker-quickstart.mdx:190]: would prefer more descriptive service names e.g. `<chainname>-validator`

File: docs/operate/docker-quickstart.mdx (hunk 1 above)
Comment (pops-one) [reply docs/operate/docker-quickstart.mdx:89]: I just pushed a new commit with a tested dockerfile

File: docs/operate/docker-quickstart.mdx (hunk 2 above)
Comment (pops-one) [reply docs/operate/docker-quickstart.mdx]: I updated the doc accordingly

**Review by paulbalaji — APPROVED** [review]

**Pull request merged**